Dockerfile

```diff
@@ -37,6 +37,17 @@ RUN curl -sL https://graphviz.gitlab.io/pub/graphviz/stable/SOURCES/graphviz.tar
     ./configure && make && make install && \
     cd - && rm -rf graphviz*
 
+ENV REDIS_VERSION=6.2.1
+RUN curl -OL https://download.redis.io/releases/redis-${REDIS_VERSION}.tar.gz && \
+    tar xzf redis-${REDIS_VERSION}.tar.gz && \
+    cd redis-${REDIS_VERSION} && \
+    make && \
+    cp ./src/redis-server /usr/local/bin/ && \
```

Comment:
No `make install` target?

Reply:
There is but the default is to `install`. I'll update just to make it clear.

```
kgann:redis-6.2.1 ⇒ pwd
/Users/kgann/Downloads/redis-6.2.1
kgann:redis-6.2.1 ⇒ cat Makefile
# Top level makefile, the real shit is at src/Makefile

default: all

.DEFAULT:
	cd src && $(MAKE) $@

install:
	cd src && $(MAKE) $@

.PHONY: install
```